localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,390 @@
+ prince_male_dragon_empire:0 "Prince"
+ princess_female_dragon_empire:0 "Princess"
+
+ ### Red Dragons ###
+ dragonprince_male_red_dragon_king:0 "Dragon Prince"
+ dragonprincess_female_red_dragon_king:0 "Dragon Princess"
+ dragonking_male_red_dragon_emperor:0 "Dragon King"
+ dragonqueen_female_red_dragon_emperor:0 "Dragon Queen"
+
+ ### Mantids ###
+ emperor_male_mantid:0 "Grand Emperor"
+ empress_female_mantid:0 "Grand Empress"
+
+ imperial_vizier_male_mantid:0 "Imperial Vizier"
+ imperial_vizier_female_mantid:0 "Imperial Vizier"
+ grand_vizier_male_mantid:0 "Grand Vizier"
+ grand_vizier_female_mantid:0 "Grand Vizier"
+ vizier_male_mantid:0 "Vizier"
+ vizier_female_mantid:0 "Vizier"
+
+ ### Qiraji ###
+ # Special
+ emperor_ahn_qiraj_male_qiraji:0 "Emperor"
+ empress_ahn_qiraj_female_qiraji:0 "Emperor"
+ empire_ahn_qiraj:0 "Empire"
+
+ # Others
+ lord_male_qiraji_count:0 "Lord"
+ lady_female_qiraji_count:0 "Lady"
+ hive_qiraji_county:0 "Hive"
+ high_overlord_male_qiraji_duke:0 "High Overlord"
+ high_overlady_female_qiraji_duke:0 "High Overlady"
+ brood_qiraji_duchy:0 "Brood"
+
+ ### Wildhammer ###
+ # Feudal
+ chief_male_wildhammer_count:0 "Chief"
+ chieftess_female_wildhammer_count:0 "Chieftess"
+ chiefdom_wildhammer_county:0 "Chiefdom"
+ thane_male_wildhammer_duke:0 "Thane"
+ thane_female_wildhammer_duke:0 "Thane"
+ high_chiefdom_wildhammer_duchy:0 "High Chiefdom"
+ high_thane_male_wildhammer_king:0 "High Thane"
+ high_thane_female_wildhammer_king:0 "High Thane"
+
+ ### Dark Iron ###
+ # Feudal
+ lord_male_darkiron_count:0 "Lord"
+ lady_female_darkiron_count:1 "Lady"
+ lordship_darkiron_county:0 "Lordship"
+ thane_male_darkiron_duke:0 "Thane"
+ thane_female_darkiron_duke:0 "Thane"
+
+ ### Old Dwarven / Frostborn ###
+ # Feudal
+ chief_male_old_dwarven_count:0 "Chief"
+ chieftess_female_old_dwarven_count:0 "Chieftess"
+ chiefdom_old_dwarven_county:0 "Chiefdom"
+ thane_male_old_dwarven_duke:0 "Thane"
+ thane_female_old_dwarven_duke:0 "Thane"
+ high_chiefdom_old_dwarven_king:0 "High Chiefdom"
+ high_thane_male_old_dwarven_king:0 "High Thane"
+ high_thane_female_old_dwarven_kingdom:0 "High Thane"
+
+ ### Vrykul and Kvaldir ###
+ # Feudal
+ thane_male_vrykul_count:0 "Thane"
+ thane_female_vrykul_count:0 "Thane"
+ chiefdom_vrykul_county:0 "Chiefdom"
+ jarl_male_vrykul_duke:0 "Jarl"
+ frue_female_vrykul_duke:0 "Frue"
+ jarldom_vrykul_duchy:0 "Jarldom"
+ high_king_male_vrykul_emperor:0 "High King"
+ high_queen_female_vrykul_emperor:0 "High Queen"
+ high_kingdom_vrykul_empire:0 "High Kingdom"
+
+ ### Frost Vrykul ###*
+ # Feudal
+ thane_male_frost_vrykul_count:0 "Thane"
+ thane_female_frost_vrykul_count:0 "Thane"
+ chiefdom_frost_vrykul_county:0 "Chiefdom"
+ overthane_male_frost_vrykul_duke:0 "Overthane"
+ overthane_female_frost_vrykul_duke:0 "Overthane"
+ jarldom_frost_vrykul_duchy:0 "Jarldom"
+ high_king_male_frost_vrykul_emperor:0 "High King"
+ high_queen_female_frost_vrykul_emperor:0 "High Queen"
+ high_kingdom_frost_vrykul_empire:0 "High Kingdom"
```

Comment:
Don't get why you haven't united vrykul under culture group.

Reply:
done in 33c3e97dc